src/styles/main.scss

```
@@ -1,7 +1,49 @@
 @import "utils";
 @import "fonts";
 @import "typography";
+@import "reset";
+@import "header";
+@import "nav";
+@import "about";
+@import "gallery";
+@import "section";
+@import "button";
+@import "consultation";
+@import "form";
+@import "name";
+@import "reviews";
+@import "logo";
+@import "card";
+@import "contacts";
+@import "footer";
+@import "mobile-nav";
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 16px;
+  scroll-behavior: smooth;
+}
 
 body {
-  background: $c-gray;
+  background: #fff;
+}
+
+.page {
+  &__first-screen {
+    height: 100vh;
+    background: url(../images/header.jpg);
+    background-size: cover;
+    background-position: center;
+    padding: 0 20px;
```

Сортування + можна об'єднати коли вказуєш кілька властивостей
```suggestion
.page {
  &__first-screen {
    height: 100vh;
    padding: 0 20px;
    
    background: url(../images/header.jpg) center / cover;
```